FSQCellManifest/FSQCellManifest.m

```diff
@@ -622,8 +622,8 @@ - (nullable NSArray *)removeCellRecordsAtIndexPaths:(NSArray *)indexPaths
             [removedCellIndexPathsMutable addObject:[self indexPathForRowOrItem:idx inSection:sectionIndex]];
         }];
     }
-    
-    NSIndexSet *sectionIndexesToRemove = [sectionIndexesToRemoveMutable copy];
+    //NOTE:init sectionIndexesToRemove here, because we must let sectionIndexesToRemove pass nil check so that managedViewUpdates(UI) will be excuted otherwise UI won't be updated
+    NSIndexSet *sectionIndexesToRemove = [sectionIndexesToRemoveMutable copy]?:[NSIndexSet new];
```

Comment:
Since this is open source code, we should be very very very clean and intelligible in how we write it. So, can you put a little more whitespace in here? For example: spaces around the `?:` so it's `] ?: [` instead. Also, spaces at the start of the comment and be sure to not leave out words: `// NOTE: We have to make sure sectionIndexesToRemove is not nil because...`